src/lib/db/feature-toggle-store.ts

```diff
@@ -377,6 +396,40 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
 
         return toggle;
     }
+
+    async markPotentiallyStaleFeatures(
+        currentTime?: string,
+    ): Promise<string[]> {
+        const query = this.db(TABLE)
+            .update('potentially_stale', true)
```

Comment:
Should we also unmark them if configuration changes?

Reply:
That should be handled in the update method,  I think. Do you disagree?